packages/oauth/__tests__/accounts-oauth.ts

```diff
@@ -0,0 +1,181 @@
+import { UserObjectType } from '@accounts/common';
+import { DBInterface } from '@accounts/server';
+import { AccountsOauth } from '../src';
+
+const user: UserObjectType = {
+  id: '1',
+  username: 'neo',
+  email: 't1@matrix.com',
+};
+const mockStore: DBInterface = {
```

Comment:
I think it be useful to move `mockStore` to a shared location to be reused by other tests in other packages. Could be left for a future PR.